nautobot/ipam/tests/test_views.py

```diff
@@ -167,6 +167,25 @@ def setUpTestData(cls):
             "remove_locations": [cls.locations[1].pk],
         }
 
+    def test_indentation_is_removed_on_filter_or_sort(self):
```

Comment:
Why a different test function for this view?

Reply:
`Prefix` is not a `TreeNode` model, hence it is not possible to query for entries with children using `children__isnull=False` in order to determine indentation has been removed. I made a separate test case for Prefix` `rather than changing the Base test case to fit it.

Reply:
It's not a `TreeModel` but it does have a `parent` FK and a `children` reverse relation.

```
>>> Prefix.objects.filter(children__isnull=True).count()
89
>>> Prefix.objects.filter(children__isnull=False).distinct().count()
28
>>> Prefix.objects.count()
117
```